src/legacy/core_plugins/data/public/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -74,6 +73,7 @@ export class IndexPattern implements StaticIndexPattern {
   private originalBody: { [key: string]: any } = {};
   private fieldsFetcher: any;
   private shortDotsEnable: boolean = false;
+  private fieldFormats: FieldFormatRegisty = getFieldFormats();
```

Comment:
we use fieldFormats in 2 places in this file. I suggest you call getFieldFormats directly in these places, rather than saving it as a class field.

Reply:
Done